genome.spec

```diff
@@ -1,34 +1,45 @@
 /*
-KBase genome ID
+Workspace reference to a genome object.
 @id kb
 */
 typedef string Genome_id;
 
 /*
-Reference to a source_id
+External database reference from which this genome data originated.
+Examples: "NC_014248"
 @id external
 */
 typedef string source_id;
 
 /*
-Structure for a publication
-(float pubmedid
-string source (ex. Pubmed)
-string title
-string web address
-string  publication year
-string authors
-string journal)
+Publication data about this genome.
+Fields:
+    0: float  pubmedid
+    1: string source (ex. Pubmed)
+    2: string title
+    3: string web address
+    4: string publication year
+    5: string authors
```

Comment:
What format?

Reply:
I added an example format from data I've looked at, but I don't know if it's standardized